pom.xml

```diff
@@ -99,50 +99,11 @@
         </dependencies>
     </dependencyManagement>
 
-    <pluginRepositories>
-        <pluginRepository>
-            <id>repo.eclipse.org</id>
-            <name>Project Repository - Releases</name>
-            <url>https://repo.eclipse.org/content/groups/cbi/</url>
-        </pluginRepository>
-        <pluginRepository>
-            <id>microprofile.repo.eclipse.org</id>
-            <name>Microprofile Project Repository - Releases</name>
-            <url>https://repo.eclipse.org/content/groups/microprofile/</url>
-        </pluginRepository>
-    </pluginRepositories>
-
     <build>
-        <pluginManagement>
-            <plugins>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-release-plugin</artifactId>
-                    <version>${version.plugin.release}</version>
-                    <configuration>
-                        <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <localCheckout>true</localCheckout>
-                        <useReleaseProfile>false</useReleaseProfile>
-                        <arguments>${arguments} -Prelease -Drevremark=${revremark}</arguments>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>org.eclipse.cbi.maven.plugins</groupId>
-                    <artifactId>eclipse-jarsigner-plugin</artifactId>
-                    <version>1.1.4</version>
-                </plugin>
-                <plugin>
-                    <groupId>com.github.eirslett</groupId>
-                    <artifactId>frontend-maven-plugin</artifactId>
-                    <version>1.6</version>
-                </plugin>
-            </plugins>
-        </pluginManagement>
         <plugins>
             <plugin>
                 <groupId>org.apache.rat</groupId>
                 <artifactId>apache-rat-plugin</artifactId>
-                <version>0.12</version>
```

Comment:
Is it possible to remove this plugin because it was in parent pom.xml? Is the excludes clause needed to be updated? If so, can you add the extra lines in the microprofile-parent repo?

Reply:
Boiled down to the:
```xml
<exclude>**/*.svg</exclude>
<exclude>**/*.json</exclude>
```

Reply:
Done eclipse/microprofile-parent/pull/57